docs/setup/production.asciidoc

```diff
@@ -125,4 +126,17 @@ elasticsearch.hosts:
 --------
 
 Related configurations include `elasticsearch.sniffInterval`, `elasticsearch.sniffOnStart`, and `elasticsearch.sniffOnConnectionFault`.
-These can be used to automatically update the list of hosts as a cluster is resized.  Parameters can be found on the {kibana-ref}/settings.html[settings page].
\ No newline at end of file
+These can be used to automatically update the list of hosts as a cluster is resized.  Parameters can be found on the {kibana-ref}/settings.html[settings page].
+
+[float]
+[[memory]]
+=== Memory
+Kibana has a default maximum memory limit of 1.4 GB, and in most cases, we recommend leaving this unconfigured.  In some scenarios, such large reporting jobs
+, it may make sense to tweak limits to meet more specific requirements.
```

Comment:
If you could move the comment to the previous line it will remove the space before the comma.